src/pages/TokenDetails/index.tsx

```diff
@@ -6,11 +6,21 @@ import { TimePeriod, toHistoryDuration, validateUrlChainParam } from 'graphql/da
 import useParsedQueryString from 'hooks/useParsedQueryString'
 import { atomWithStorage, useAtomValue } from 'jotai/utils'
 import { useEffect, useMemo, useState } from 'react'
+import { Helmet } from 'react-helmet'
 import { useParams } from 'react-router-dom'
 import { getNativeTokenDBAddress } from 'utils/nativeTokens'
 
 export const pageTimePeriodAtom = atomWithStorage<TimePeriod>('tokenDetailsTimePeriod', TimePeriod.DAY)
 
+const getTitle = (tokenQuery: ReturnType<typeof useTokenQuery>['data']) => {
+  const tokenName = tokenQuery?.token?.name
+  const tokenSymbol = tokenQuery?.token?.symbol
+  if (!tokenName && !tokenSymbol) {
+    return 'Buy & Trade: Live Price & Chart on Uniswap'
+  }
+  return `Buy & Trade ${tokenName ?? ''} ${tokenSymbol ? `(${tokenSymbol})` : ''}: Live Price & Chart on Uniswap`
```

Comment:
would it be weird if theres a symbol but not a name and it shows up like

`Buy & Trade (USDC): Live Price & Chart on Uniswap`

Reply:
oh true ... let me change that